resources/function_help/json/get_current_form_field_value.txt

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "current_value",
+  "type": "function",
+  "description": "Returns the current value of a field in the form or table row currently being edited.",
```

Comment:
I'd probably expand slightly on this for clarification:

"Returns the current, unsaved value of a field in the form or table row currently being edited. This will differ from the feature's actual attribute values for features which are currently being edited or have not yet been added to a layer."